packages/perseus/src/widgets/__tests__/radio/choice-icon_test.jsx

```diff
@@ -63,5 +63,52 @@ describe("choice icon", () => {
                 });
             },
         );
+
+        describe.each([[true], [false]])("correct: %s", (correct) => {
+            it("displays correctness as expected", () => {
+                if (product === "library") {
```

Comment:
I was able to separate these into two tests, but do you think I should just separate out the true/false wrapper and have a test for when correct is true and a separate test for when it is false since I am expecting two different things for each?

Reply:
I would suggest 4 distinct tests, I think.  There's definitely _some_ shared logic, but I think the if/else branching is more of an issue than some code duplication. 

So yes, I'd separate out the true/false wrapper. 👍